src/hs/testlib/Test/Expr.hs

```diff
@@ -61,14 +61,14 @@ testMain name typeOrder pat degen = testMainEx $ TestConfig conf
 
 testMainEx :: TestConfig -> Q [Dec]
 testMainEx config = do
-    sname' <- lookupValueName sn
-    $(pfail "Could not find student expression %s") name & when (isNothing sname')
-    tname   <- lookupValueName tn
+    studentExp' <- maybe unitExp (fmap (fmap VarE) . lookupValueName) sn
+    $(pfail "Could not find student expression %s") name & when (isNothing studentExp')
+    tname   <- maybe (pure Nothing) lookupValueName tn
```

Comment:
nestačí tady `lookupValueName <$> tn`?

Reply:
Ne, potřebuju i to `Nothing` zabalit do `Q`.

Reply:
ah, dvě monády